File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int count = 0;
        for(String word : words){
            boolean istrue = true;
            for(char ch: word.toCharArray()){
                boolean found = false;
                for(char c: allowed.toCharArray()){
                    if(ch == c){
                        found = true;
                        // break;
                    }
                }
                if(!found){
                    istrue = false;
                    // break;
                }
            }
            if(istrue){
                count++;
            }
        }
        return count;
    }
}

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let count = 0;
    for(let word of words){
        let isTrue = true;
        for(let ch of word){
            let found = false;
            for(let c of allowed){
                if(ch === c){
                    found = true;
                    break;
                }
            }
            if(!found){
                isTrue = false;
                break;
            }
        }
        if(isTrue){
            count++;
        }
    }
    return count;
};